client/components/app.js

```diff
@@ -39,12 +46,20 @@ class App extends React.Component {
     }
   }
 
+  updateQuantityChoice(choice) {
+    this.setState({quantityChoice: Number(choice)});
+  }
+
+  updatePersonalizationChoice(choice) {
+    this.setState({personalizationChoice: choice});
+  }
+
   updateState(newData) {
     this.setState(newData);
   }

```

Comment:
Never used setState like this without passing an object in, not sure if its good practice. I assume newData has the same properties as your state does and it changes all those values together? But if it works as intended then that's fine, I just think it could hurt later on if you add more properties in state.

Reply:
This will just update the properties in state that are in newData. Any other properties on state,
will be left as they were, I just found out that setState works that way this morning.